### ISSUE TITLE
Sticky Header feature

### ISSUE THREAD

**Issue by arya312** [onesoft-sudo/tes-website#9]
Sticky header feature could be introduced for ease of going to any page of the website. The header could remain at the same position even while scrolling.

**Comment by virtual-designer** [comment]
Once again thanks for posting this issue! I will implement this soon.

**Comment by arya312** [comment]
No problem! The project looks intriguing to me, is there any chance I could help with the same? I am taking part in Hacktoberfest and so I was thinking I could make some pull requests myself for building the project!

**Comment by virtual-designer** [comment]
> No problem! The project looks intriguing to me, is there any chance I could help with the same? I am taking part in Hacktoberfest and so I was thinking I could make some pull requests myself for building the project!

Yes sure! If you want I can assign you to this issue and then you can implement the sticky header feature!

**Comment by arya312** [comment]
Cool, that'd be great! I would love to work on this issue.

**Comment by virtual-designer** [comment]
> Cool, that'd be great! I would love to work on this issue.

Assigned! Good luck :)
